src/thunderbots/shared/constants.h

```diff
@@ -16,17 +16,22 @@ const double BALL_MAX_RADIUS = 0.0215;
 // The maximum speed achievable by our robots, in metres per second.
 // TODO: Determine a more realistic value
 const double ROBOT_MAX_SPEED = 2.0;
+// The maximum angular speed achievable by our robots, in rad/sec
+// TODO: Determine a more realistic value
+const double ROBOT_MAX_ANG_SPEED = 4.0;
 // The maximum acceleration achievable by our robots, in metres per seconds squared.
 // TODO: Determine a more realistic value
 const double ROBOT_MAX_ACCELERATION = 3.0;
-
+// The maximum angular acceleration achievable by our robots, in radians per second
+// squared
+// TODO: Determine a more realistic value
+const double ROBOT_MAX_ANG_ACCELERATION = 6.0;
 /* Unit Conversion */
 const double MILLIMETERS_PER_METER = 1000.0;
 const double METERS_PER_MILLIMETER = 1.0 / 1000.0;
 
-const double CENTIRADIANS_PER_RADIAN = 100.0;
-const double RADIANS_PER_CENTIRADIAN = 1.0 / 100.0;
-
+const double CENTIRADIANS_PER_RADIAN      = 100.0;
+const double RADIANS_PER_CENTIRADIAN      = 1.0 / 100.0;
```

Comment:
super nit: A newline below this line is nice because it separates the "angle/radian" conversions from the "time" conversions.

Reply:
Added space.